src/Common/src/Microsoft/Win32/SafeHandles/GssSafeHandles.cs

```diff
@@ -78,6 +78,14 @@ internal class SafeGssCredHandle : SafeHandle
         /// </summary>
         public static SafeGssCredHandle Create(string username, string password, bool isNtlmOnly)
         {
+            if (isNtlmOnly && !Interop.NetSecurityNative.IsNtlmInstalled())
```

Comment:
Can the IsNtlmInstalled call be cached?

Reply:
Yes.  I can probably cache this at the .NET layer with a Lazy<T>.